contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -1869,6 +1942,18 @@ PrepareRowDescription(TupleDesc typeinfo, List *targetlist, int16 *formats,
 					SetColMetadataForTimeType(col, TDS_TYPE_DATETIMEOFFSET, atttypmod);
 				}
 				break;
+			case TDS_SEND_GEOMETRY:
+				/*
+				* TODO: Check TDS versioning requirements 
```

Comment:
does this comment need to stay here?

Reply:
Since we are setting column metadata just like binary type so there shouldn't be any dependency on TDS version, so I have removed the comment